### ISSUE TITLE
JSe.Array does not fulfll JS.ArrayConstructor interface

### ISSUE THREAD

**Issue by absolutejam** [Shmew/Fable.Extras#2]
## Description

Just added `Fable.Extras` as per your suggestion in Slack, but I'm hitting the following:

```
/Users/james/Documents/slate-fs/src/Slate/.fable/Fable.Extras.0.4.0/Extras.fs(620,19): (620,38) error FSHARP: No implementation was given for those members: 
        'abstract member JS.ArrayConstructor.Create : size:int -> 'T []'
        'abstract member JS.ArrayConstructor.from : arg:obj -> 'T []'
Note that all interface members must be implemented and listed under an appropriate 'interface' declaration, e.g. 'interface ... with member ...'. (code 366)
```

## Steps to reproduce

Add package and attempt to run Fable.

## Additional context

```
dotnet fable --version
Fable: F# to JS compiler 3.1.5
```

`paket.lock` shows Fable.Core version:
```
    Fable.Core (3.2.4)
```

**Comment by absolutejam** [comment]
Can these extra members just be added with `jsNative`?

**Comment by Shmew** [comment]
This should be fixed with 0.4.1
